libraries/lib-wave-track/WaveClip.h

```diff
@@ -611,6 +615,9 @@ class WAVE_TRACK_API WaveClip final :
    double mTrimRight { 0 };
    //! @}
 
+   std::weak_ptr<PitchShiftChangePublisher> mPitchShiftChangePublisher;
+   double mSemitoneShift { 0 };
```

Comment:
This commit is strange.

As I said I don't like it that lib-wave-track mentions extraneous things like quarter notes and stretch ratios, and now this is one more thing.  I might figure out a restructuring in my branch.

Why can't a track or clip simply publish whenever it changes, and have subscribers attached to it?  Why must you first set a publishing interface into it?

Reply:
Looks good to me - see "Use Observer framework" commit